packages/element/src/test/create-interpolate-element.js

```diff
@@ -209,15 +209,13 @@ describe( 'createInterpolateElement', () => {
 		};
 		const { container, rerender } = render( <TestComponent switchKey /> );
 
-		expect( container.firstChild ).toContainHTML( '<em>string!</em>' );
-		expect( container.firstChild ).not.toContainHTML( '<strong>' );
+		expect( container ).toContainHTML( '<em>string!</em>' );
```

Comment:
I thought we aim to remove the usage of `container`, no?

Reply:
We're just fine with `container` on its own, as long as we don't do DOM traversing or raw DOM queries. Actually, RTL doesn't recommend any better way to verify that a particular HTML has been rendered, so this is good as it is.